vault/init.go

```diff
@@ -173,8 +173,13 @@ func (c *Core) Initialize(barrierConfig, recoveryConfig *SealConfig) (*InitResul
 	}()
 
 	// Perform initial setup
+	if err := c.setupCluster(); err != nil {
```

Comment:
It's actually on purpose; even if HA is off we still want to generate
a cluster-id and persist it. *That said* if HA is off I will have it
skip key/cert generation.